packages/outstatic/tailwind.config.js

```diff
@@ -12,6 +12,16 @@ module.exports = {
       colors: {
         brandblue: colors.blue[500],
         brandred: colors.red[500]
+      },
+      animation: {
+        draw: 'draw 3000ms linear infinite'
+      },
+      keyframes: {
+        draw: {
+          '0%': { 'stroke-dashoffset': '0' },
+          '50%': { 'stroke-dashoffset': '4000' },
+          '100%': { 'stroke-dashoffset': '0' }
+        }
```

Comment:
```suggestion
      animation: {
        draw: 'draw 5s cubic-bezier(0,.57,1,.43) infinite'
      },
      keyframes: {
        draw: {
          '0%': { 'stroke-dashoffset': '0' },
          '49.999%': { 'stroke-dashoffset': '-3000' },
          '50%': { 'stroke-dashoffset': '4000' },
          '50.001%': { 'stroke-dashoffset': '4000' },
          '100%': { 'stroke-dashoffset': '0' }
        }
      }
```

Reply:
Making the line only move forward here, also adding some cubic-bezier for better physics 🤓